src/sandboxes/web3-react-v6-sandbox/App.tsx

```diff
@@ -0,0 +1,238 @@
+ */
+import React, { useState, useCallback } from 'react';
+import styled from 'styled-components';
+
+import { TLog } from './types';
+import { Logs, Sidebar } from './components';
+
+import { useWeb3React, Web3ReactProvider } from '@web3-react/core';
+import { Web3Provider } from '@ethersproject/providers';
+import { PhantomConnector } from "web3-react-v6-phantom"
+import { Signer } from 'ethers';
+import { parseUnits } from 'ethers/lib/utils';
+
+const phantom = new PhantomConnector({
+  supportedChainIds: [1, 5],
+});
+
+// =============================================================================
+// Styled Components
+// =============================================================================
+
+const StyledApp = styled.div`
+  display: flex;
+  flex-direction: row;
+  height: 100vh;
+  @media (max-width: 768px) {
+    flex-direction: column;
+  }
+`;
+
+// =============================================================================
+// Constants
+// =============================================================================
+
+const message = 'To avoid digital dognappers, sign below to authenticate with CryptoCorgis.';
+const TX = {
+  to: '0x0000000000000000000000000000000000000000',
+  value: parseUnits('1', 'wei'),
+  chainId: 0x5, // Goerli network
+};
+// =============================================================================
+// Typedefs
+// =============================================================================
+
+export type SidebarMethods =
+  | {
+      name: string;
+      onClick: () => Promise<string>;
+    }
+  | {
+      name: string;
+      onClick: () => Promise<void>;
+    }
+  | {
+      name: string;
+      onClick: () => void;
+    };
+
+interface Props {
+  // connectedMethods: ConnectedMethods[];
+  logs: TLog[];
+  clearLogs: () => void;
+  createLog: (log: TLog) => void;
+}
+
+// =============================================================================
+// Hooks
+// =============================================================================
+
+/**
+ * @DEVELOPERS
+ * The fun stuff!
+ */
+const useProps = (): Props => {
+  const [logs, setLogs] = useState<TLog[]>([]);
+
+  const createLog = useCallback(
+    (log: TLog) => {
+      return setLogs((logs) => [...logs, log]);
+    },
+    [setLogs]
+  );
+
+  const clearLogs = useCallback(() => {
+    setLogs([]);
+  }, [setLogs]);
+
+  return {
+    logs,
+    createLog,
+    clearLogs,
+  };
+};
+
+// =============================================================================
+// Stateless Component
+// =============================================================================
+
+const StatelessApp = React.memo((props: Props) => {
+  const { logs, clearLogs, createLog } = props;
+  const { library, activate, deactivate } = useWeb3React();
+
+  const handleDisconnect = () => {
+    try {
+      deactivate();
+      createLog({
+        status: 'warning',
+        method: 'disconnect',
+        message: 'user disconnected wallet',
```

Comment:
Let's stay consistent on capitalization and message content for all our Sandbox methods. See multichain sandbox for example https://github.com/0xproflupin/phantom-sandboxes/blob/637c218208f1ab62ab033fd2eaea9f6a18e68a02/src/sandboxes/multi-chain-sandbox/App.tsx#L144